gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/main/java/io/gravitee/rest/api/service/migration/APIV1toAPIV2Converter.java

```diff
@@ -15,15 +15,7 @@
  */
 package io.gravitee.rest.api.service.migration;
 
-import static io.gravitee.common.http.HttpMethod.CONNECT;
-import static io.gravitee.common.http.HttpMethod.DELETE;
-import static io.gravitee.common.http.HttpMethod.GET;
-import static io.gravitee.common.http.HttpMethod.HEAD;
-import static io.gravitee.common.http.HttpMethod.OPTIONS;
-import static io.gravitee.common.http.HttpMethod.PATCH;
-import static io.gravitee.common.http.HttpMethod.POST;
-import static io.gravitee.common.http.HttpMethod.PUT;
-import static io.gravitee.common.http.HttpMethod.TRACE;
+import static io.gravitee.common.http.HttpMethod.*;
```

Comment:
:'(

Reply:
:aa: where is the PR with the config file ?